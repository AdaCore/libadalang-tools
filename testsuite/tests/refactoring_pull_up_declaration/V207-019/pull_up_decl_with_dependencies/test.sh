pull_up_declaration -P default.gpr --source main.adb --line 18 --column 12
