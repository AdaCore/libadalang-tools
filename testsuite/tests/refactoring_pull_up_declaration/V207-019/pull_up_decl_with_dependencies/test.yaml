title: pull_up_decl_with_dependencies

description: pull_up_declaration tool test

driver: shell_script
