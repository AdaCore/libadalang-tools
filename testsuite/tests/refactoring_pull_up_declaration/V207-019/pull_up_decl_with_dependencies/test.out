main.adb
4:1-4:1    type My_Natural_Range;

   --  Some comments
   --  A very natural range
   type My_Natural_Range is range 0 .. 10;
   --  A very natural Natural

type My_Natural is new Natural range 0 .. 20; --  Some comments
   --  A very natural array of very natural Naturals
   type My_Natural_Array is array (My_Natural_Range) of My_Natural;
   --  A very natural array of very natural Naturals


8:1-14:1 
16:45-20:1 

